package.json

```diff
@@ -33,7 +33,7 @@
     "cz-ovh-commit": "^1.0.0",
     "less": "^2.7.2",
     "less-plugin-clean-css": "^1.5.1",
-    "ovh-ui-kit": "git+https://github.com/ovh-ux/ovh-ui-kit.git#v1.0.0-alpha.20",
+    "ovh-ui-kit": "git+https://github.com/ovh-ux/ovh-ui-kit.git#af914a579f2dacc819123ece341a5455a59f16fa",
```

Comment:
Does it will be released soon?

Reply:
Yes, it's related with this PR: https://github.com/ovh-ux/ovh-ui-kit/pull/121
To avoid to set it to master. It'll be changed later.